jumpstarter/states.py

```diff
@@ -48,13 +59,15 @@ def __init__(self, actor_state=ActorState):
                 actor_state.starting.value.resources_acquired,
                 actor_state.starting.value.tasks_started,
             ],
-            trigger="start",
+            trigger=ActorStateTriggers.start,
```

Comment:
How does this work?

According to the documentation, the trigger must be a string.

Reply:
This is a super convenient pattern in _some_ places. It gives you the convenience of treating you enum as a `str` where needed. But, I believe you lose out on some ordering.

```python
class Foo(str, Enum):
    ...
```

Reply:
I understand now.